clients/privacy-center/cypress/e2e/consent-notices.cy.ts

```diff
@@ -118,6 +118,10 @@ describe("Privacy notice driven consent", () => {
         expect(
           preferences.map((p: ConsentOptionCreate) => p.preference)
         ).to.eql(["opt_in", "opt_in", "acknowledge"]);
+        // Wait for toast so that we know cookie is ready to be inspected
```

Comment:
unrelated to this issue, but the test failed on this PR so I added this which should make the test less flaky